pkg/capabilities/consensus/ocr3/common_builders_generated.go

```diff
@@ -6,6 +6,50 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/workflows"
 )
 
+type EncoderCap workflows.CapDefinition[Encoder]
+
+type EncoderConfigCap interface {
```

Comment:
This is probably specific to EVM, no?

Reply:
Yeah, good call, I'll make it take a map[string]any.